en-US/emails/2017/FxnU_oct.lang

```diff
@@ -0,0 +1,150 @@
+
+
+;How? Because by “us,” we mean not just Firefox, but Mozilla, the technology nonprofit that backs us. Mozilla brings internet education to people all over the world, fights for your online rights and keeps corporate power in check.
+How? Because by “us,” we mean not just Firefox, but Mozilla, the technology nonprofit that backs us. Mozilla brings internet education to people all over the world, fights for your online rights and keeps corporate power in check.
+
+
+;You’re already doing good. And together we can do more. Learn how you can rock the power of your internet citizenship.
+You’re already doing good. And together we can do more. Learn how you can rock the power of your internet citizenship.
+
+
+;Thanks,
+Thanks,
+
+
+;The Firefox Team
+The Firefox Team
+
+
+;P.S. If you have any questions, we’re always here to help.
+P.S. If you have any questions, we’re always here to help.
+
+
+# Email 2
+# Subject Line
+;Goodbye corporate domination
+Goodbye corporate domination
+
+
+# Preheader
+;Hello healthier web.
+Hello healthier web.
+
+
+# Headline
+;“All browsers are the same, right?”
+“All browsers are the same, right?”
+
+
+;Not quite. Firefox is on a mission — to be faster and leaner than ever, always respectful of your privacy and open like no other browser out there. Because it’s built for people, not profit. Firefox is part of Mozilla’s mission to keep you in control of your online experience and ensure that the web stays weird, healthy and inviting to all.
+Not quite. Firefox is on a mission — to be faster and leaner than ever, always respectful of your privacy and open like no other browser out there. Because it’s built for people, not profit. Firefox is part of Mozilla’s mission to keep you in control of your online experience and ensure that the web stays weird, healthy and inviting to all.
+
+
+;When you browse with Firefox, you’re supporting a nonprofit organization that fights for net neutrality, promotes sensible copyright reform in the EU, teaches web literacy around the world and keeps a close eye on the overall health of the internet.
+When you browse with Firefox, you’re supporting a nonprofit organization that fights for net neutrality, promotes sensible copyright reform in the EU, teaches web literacy around the world and keeps a close eye on the overall health of the internet.
+
+
+;We’re here to help you go your own way online and give you the power to browse freely, without being tracked or profited from.
+We’re here to help you go your own way online and give you the power to browse freely, without being tracked or profited from.
+
+
+;Because you are not a product. You’re an explorer on the open web. You deserve something different. Something better.
+Because you are not a product. You’re an explorer on the open web. You deserve something different. Something better.
+
+
+;We’ve got your back,
+We’ve got your back,
```

Comment:
Why the comma? What's following?

Reply:
This is the salute at the end of a message. We can add "<br> The Firefox Team with each of these messages.

Reply:
Makes sense, as long as people doing copy and paste understand what they're supposed to do.